products/green_fast_track/bash/export.sh

```diff
@@ -90,15 +90,18 @@ mkdir -p output && (
     export_source sources__nyc_historic_buildings_points MULTIPOINT
     export_source sources__nyc_historic_buildings_lots
     export_source sources__nyc_historic_buildings_buffers
+    export_source sources__nyc_historic_buildings_buffers_200
```

Comment:
@damonmcc do you know if these exports are needed specifically, or if just the shadow buffer on line 104 is needed? I imagine that was part of your conversation leading to #826

Reply:
only the shadow buffer is needed. but no worries on getting these source geometry export exactly right in this PR!

Jack and I are gonna touch base next week to decide what would be ideal for these exports

Reply:
Cool, I'll leave this then